Tests/Tests.XamarinMac/Tests.XamarinMac.csproj

```diff
@@ -57,7 +57,7 @@
     <AOTMode>None</AOTMode>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(RunConfiguration)' == 'Default' ">
-    <StartArguments>--headless --labels=After</StartArguments>
+    <StartArguments>--labels=After --where=test=~AAAATests</StartArguments>
```

Comment:
This is something you need to run tests, but you should not push it, because we actually use it also to run tests automatically.

Reply:
Remember about this